CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## release note
 
+## [v3.1.0](https://github.com/skyway/skyway-js-sdk/releases/tag/v3.1.0) - 2020-10-09
+
+### Fixed
```

Comment:
`Deprecated` is better than `Fixed`
Please refer to this https://keepachangelog.com/en/1.0.0/